flow-decomposition/src/main/java/com/powsybl/flow_decomposition/NetworkStateManager.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.flow_decomposition;
+
+import com.powsybl.iidm.network.Network;
+
+/**
+ * @author Hugo Schindler {@literal <hugo.schindler at rte-france.com>}
+ */
+class NetworkStateManager {
+    private final Network network;
+    private final VariantManager variantManager;
+
+    public NetworkStateManager(Network network, XnecProvider xnecProvider) {
+        this.network = network;
+        LossesCompensator.addZeroMWLossesLoadsOnBuses(network);
```

Comment:
If no losses compensation is done, this is useless

Reply:
fixed